### ISSUE TITLE
addon cannot be updated

### ISSUE THREAD

**Issue by az20101-re** [LyubomirT/c.ai-addons#18]
![insanity](https://github.com/LyubomirT/c.ai-addons/assets/66862056/6f4b7ecc-4291-497a-b0ab-8cdadbb1aee3)

each time I try to update the addon/extension, it just stays on version 0.5.0
I tried reinstalling, restarting windows, reinstalling browser, tried on mozilla and ungoogled chromium. All the features are just stuck on 0.5.0 version. I tried also installing other outdated versions, below 0.5.0, and they worked just fine.

in short - 0.7.8 (latest) version does not works, and for some reason is still an outdated version, by comparing functions.

**Comment by az20101-re** [comment]
![sanity](https://github.com/LyubomirT/c.ai-addons/assets/66862056/c3deacfd-92a8-4fb0-ad86-7177373192f8)
should have added this image too

**Comment by LyubomirT** [comment]
It seems like in the 0.7.8 the version number remained the same in the popup, making it confuse the version checker. I'm going to release a 0.7.9 patch real quick with this resolved, please hold on for a moment. Thank you for reporting this issue.

**Comment by LyubomirT** [comment]
@az20101-re You can install the 0.7.9 patch to get the issue resolved, it should work properly now.

**Comment by az20101-re** [comment]
![изображение](https://github.com/LyubomirT/c.ai-addons/assets/66862056/8498040b-5211-4c97-80ba-fedf2cc01597)
amaze
